ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -219,6 +219,24 @@ objects:
         trap 'touch /tmp/shared/exit' EXIT
         trap 'kill $(jobs -p); exit 0' TERM
 
+        function fips_check() {
+          oc --insecure-skip-tls-verify --request-timeout=60s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}' > /tmp/nodes
```

Comment:
`-o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}'` → `-o name`.

Reply:
thanks! :)

Reply:
well, this gives `node/nodename` so have to trim